usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -155,6 +161,17 @@ def configure_spark_session(
         # conf.set("spark.hadoop.fs.s3a.committer.staging.conflict-mode", "fail")
         # conf.set("spark.hadoop.fs.s3a.committer.staging.tmp.path", "tmp/staging")
 
+        # Turn on Hive support to use a Derby filesystem DB as the metastore DB for tracking of schemas and tables
+        enable_hive_support = True
```

Comment:
So when `CONFIG.USE_AWS` is set to `True`, the input parameter value is ignored and `enable_hive_support` is set to `True`?